resources/js/components/FormField.vue

```diff
@@ -33,13 +33,18 @@
 			}
 		},
 
+        created() {
+		    Nova.$on('nova-dependency-container-' + this.field.attribute, this.dependencyChange)
```

Comment:
What does this accomplish?

Reply:
it used for another components to call this component field